wiki/People/Project_Loved_Team/en.md

```diff
@@ -0,0 +1,47 @@
+# Project Loved Team
+
+The **Project Loved Team** are members of the osu! community who manage [Project Loved](/wiki/Project_Loved), the voting platform that promotes beatmaps to the [Loved](/wiki/Beatmaps#loved) category.
+
+## Roles and Responsibilities
+
+Most of the Project Loved Team consists of **captains** for each of the [Game Modes](/wiki/Game_Modes). Captains are responsible for nominating beatmaps for the Loved category. They also write comments about the nominated maps to help readers get a feel for each map in case they have not seen or played them before.
+
+The rest of the team helps with quality control, maintenance, and organisation.
+
+## Team Members
+
+| Name | Role |
+| :-- | :-- |
+| ![][flag_CL] [-Kazu-](https://osu.ppy.sh/users/920861) | osu!taiko captain |
+| ![][flag_GB] [-Roxas](https://osu.ppy.sh/users/1986262) | osu! captain |
+| ![][flag_US] [BeasttrollMC](https://osu.ppy.sh/users/3171691) | osu! captain |
+| ![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050) | osu! captain |
+| ![][flag_US] [clayton](https://osu.ppy.sh/users/3666350) | News and forum post writer, Mapper contact |
+| ![][flag_US] [DigitalHypno](https://osu.ppy.sh/users/4384207) | osu! captain |
+| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | Beatmap updater (when others cannot!) |
```

Comment:
```suggestion
| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | Beatmap updating |
```

Reply:
using nouns instead of verbs for these roles but I will remove that extra note